apps/web/src/components/Player.tsx

```
@@ -11,22 +11,15 @@ export function Player({ summoner }: PlayerProps) {
-        <img
-          src={`/images/perk/${summoner.perks.perkStyle}.png`}
-          alt="perk"
-          className="w-4 h-4"
-        />
-        <img
-          src={`/images/perk/${summoner.perks.perkSubStyle}.png`}
-          alt="perk"
-          className="w-4 h-4"
-        />
+        {summoner.spells.map((spell) => (
+          <img src={spell.image.square} alt="perk" className="w-4 h-4" />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)